.github/workflows/ci.yml

```diff
@@ -152,7 +152,7 @@ jobs:
       matrix:
         python-version: ["3.10"]  # 3.12 stable is tested in unittest_report stage.
         db-backend: ["postgresql"]
-        nautobot-version: ["stable"]
+        nautobot-version: ["2.4"]
```

Comment:
This is going to the 3.0 branch now. Please pull in the latest changes from develop and change these back to stable.